### PR TITLE
ONL-4675 -> migrated tooltip and re-styled the story using tw classes

### PR DESCRIPTION
**Pull request by MaruaEl**
ONL-4675 -> [https://fxsolutions.atlassian.net/browse/ONL-4675](https://fxsolutions.atlassian.net/browse/ONL-4675)

- migrated tooltip styles to use tw
- re-styled story to use tw classes - is a bit different from before but I thought that is still make sense
- please note that I'm still running the regression tests on my local in case deployment fails.

### CODE REVIEW

**Comment by matallo** [comment]
needs work

Screenshot 2020-07-22 at 10 53 49

vs

Screenshot 2020-07-22 at 10 53 42

**Comment by MaruaEl** [comment]
Hi Carlos, 
thank you - I'll fix it. My bad, I thought that it didn't need to be exactly the same but to just showcase the functionality. 
Will update the pr asap

File: src/styles/components/v-tooltip.css [hunk 1]
```diff
@@ -1,6 +1,111 @@
...
+      @apply tw-bg-gray-8;
+      @apply tw-text-gray-3;
+      @apply tw-border tw-border-solid tw-border-gray-6;
+    }
+  }
+
+  &__arrow {
+    @apply tw-z-tooltip;
+    @apply tw-w-0;
+    @apply tw-h-0;
```
Comment (jmesao) [inline src/styles/components/v-tooltip.css:29]: @MaruaEl You could group tw-h-0 and put it in the same line with tw-w-0

File: src/styles/components/v-tooltip.css [hunk 2]
```diff
@@ -1,6 +1,111 @@
...
+    @apply tw-border-solid;
+    @apply tw-absolute;
+    @apply tw-border-gray-2;
```
Comment (jmesao) [inline src/styles/components/v-tooltip.css:32]: @MaruaEl You could group this and put it in the same line with tw-border-solid

File: src/styles/components/v-tooltip.css (hunk 1 above)
Comment (MaruaEl) [reply src/styles/components/v-tooltip.css]: Thanks will do! v

**Comment by matallo** [comment]
now we're talking

File: postcss.config.js [hunk 3]
```diff
@@ -28,6 +28,10 @@ module.exports = ({ file }) => {
         /data-v-.*/,
         /^ec-/,
       ],
+      whitelistPatternsChildren: [
+        /^ec-/,
+        /^ebo-/,
```
Comment (mcibique) [inline postcss.config.js:33]: We have to do this in ebo repo. remove the ebo- bit from here. chameleon has no clue about EBO.

**Pull request merged**